test/checks/forms/fieldset.js

```diff
@@ -4,19 +4,11 @@ describe('fieldset', function () {
 	var shadowSupport = axe.testUtils.shadowSupport.v1;
 	var fixtureSetup = axe.testUtils.fixtureSetup;
 
-	var checkContext = {
-		_data: null,
-		data: function (d) {
-			this._data = d;
-		},
-		relatedNodes: function (nodes) {
-			this._relatedNodes = Array.isArray(nodes) ? nodes : [nodes];
```

Comment:
Should we make `MockCheckContext` do the same conditional assignment as this is doing?

Reply:
It does do it–that's how I got the tests to pass